test/test_package.py

```diff
@@ -253,3 +253,186 @@ def test_database_from_paths(filesystem):
     db = PackageDatabase.from_paths(filesystem, paths)
     assert len(db) == len(paths)
     assert set(db) == {'angles', 'tf2', 'tf2_msgs', 'tf2_py', 'tf2_ros'}
+
+
+@pytest.mark.parametrize('sut', ['turtlebot3-ros2'], indirect=True)
+def test_package_location_ros2(sut):
+    expected_paths = {'/ros_ws/install/pcl_conversions',
```

Comment:
Does this include `/opt/ros` packages?

Reply:
It should. The `/ros_ws/install` directory includes files the packages in both `/opt/ros` and the `catkin workspace`